internal/cluster/validator.go

```diff
@@ -394,3 +396,40 @@ func (v *clusterValidator) printNetworkPrefixValid(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) validationStatus {
+	var min int64
```

Comment:
If inventory is missing for a host then this validation should be pending

Reply:
but for which host? it is not host it is cluster validation